README.md

```diff
@@ -99,103 +62,60 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Use database migration files to maintain database schema.**
+- **Build a webapp that requires the user to log in.**
+- **Use devise gem for authentication.**
+- **Use cancancan gem for authorization.**
+- **Follow wireframe for design.**
+- **Deploy the app**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
```

Comment:
- [ ] `Required`
- [ ] Amazing presentation of the project 👏 
- [ ] However, you missed out on a tiny  🤏 detail, you was supposed to highlight a piece of the code that you find interesting or is proud of making.
- [ ] Please try to make changes 🥂